packages/core/strapi/bin/strapi.js

```diff
@@ -127,7 +127,7 @@ program
 
 // `$ strapi generate:template <directory>`
 program
-  .command('generate:template <directory>')
+  .command('templates:generate <directory>')
```

Comment:
@pwizla  FYI

Reply:
Is there a reason why it's plural now?  Can't remember if that was mentioned yesterday.  Also I just remembered that originally I had used `export template`.   If there is a conflict with `generate` maybe we could use `export` not sure if that is more confusing though 🤷‍♂️ .

Reply:
I think it was to match [all of the other commands](https://github.com/strapi/strapi/blob/a633ae57ff9083a4bd1a2c1a17fb7009501a1f21/packages/core/strapi/bin/strapi.js#L186-L214) that are plural as well. This way we can also make room for future commands that may involve several templates in the future, like `templates:list` for example

Reply:
gotcha that makes sense, thanks!